web/controllers/char_controller.ex

```diff
@@ -37,4 +37,15 @@ defmodule Entice.Web.CharController do
       message: "Char created.",
       character: char})
   end
+
+  @doc "Gets the account id of the character named :name ."
+  def get_account(conn, %{"name" => :name}) do
```

Comment:
This won't work because the `:something` notation stands for an atom, which is a static string in elixir. If you want to bind whatever is in `name` to a variable, you will have to remove the colon... same goes for the `:name` further down